src/libYARP_OS/src/PortCoreInputUnit.cpp

```diff
@@ -164,12 +164,12 @@ void PortCoreInputUnit::run() {
             //printf("HAVE A REFERENCE\n");
             if (localReader!=YARP_NULLPTR) {
                 bool ok = localReader->read(br);
-                if (!br.isActive()) { done = true; break; }
```

Comment:
Is this `done` variable actually used?
There are other few `done = true; break;` in the code that are not removed.

I wonder if for readability of the code it would be better to change all of them to `done = true; continue;` instead...